thirdparty/CMakeLists.txt

```diff
@@ -87,3 +87,37 @@ if ( NOT fmt_FOUND )
     # Group all the ThirdParty items under a ThirdParty folder in IDEs
     set_target_properties(fmt PROPERTIES FOLDER "ThirdParty")
 endif()
+
+FetchContent_Declare(imgui
+    GIT_REPOSITORY
+        "https://github.com/ocornut/imgui.git"
+    GIT_TAG
+        v1.89.7
+)
+FetchContent_MakeAvailable(imgui)
+
+FetchContent_GetProperties(imgui)
+if (NOT imgui_POPULATED)
+  FetchContent_Populate(imgui)
```

Comment:
afaik you don't need to do this anymore. You just do MakeAvailable no need for populate or getproperties